app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,19 @@
+class SessionsController < ApplicationController
+  def new; end
+
+  def create
+    user = User.find_by email: params[:session][:email].downcase
+    if user&.authenticate(params[:session][:password])
```

Comment:
`&. ` code này nghĩa là gì em ơi

Reply:
nghĩa là nếu là thằng user có cùng email và cũng thằng user đó có cùng pass thì cho login

Reply:
tào lao, tìm hiểu lại link: http://mitrev.net/ruby/2015/11/13/the-operator-in-ruby/